cmd/webrpc-gen/main.go

```diff
@@ -60,13 +62,35 @@ func main() {
 		os.Exit(1)
 	}
 
+	if strings.Trim(*serviceFlag, " ") != "" {
+		s = schema.MatchServices(s, strings.Split(*serviceFlag, ","))
+	}
+
+	if strings.Trim(*ignoreFlag, " ") != "" {
```

Comment:
btw why do we trim, do you expect someone to pass `-cli=" arg "`?

Reply:
I was being super paranoid I guess. :D Ok I will remove it